qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -78,20 +78,32 @@ def run(self, qobj):
 
         return job
 
-    def properties(self, refresh=False):
+    def properties(self, refresh=False, datetime_filter=None):
         """Return the online backend properties.
 
         Args:
             refresh (bool): if True, the return is via a QX API call.
                 Otherwise, a cached version is returned.
+            datetime_filter (datetime.datetime): By specifying a filter,
+                this function returns an instance of the BackendProperties whose
+                timestamp is closest to, but less than the specified datetime
+                filter.
```

Comment:
1) For consistency, `By` should be lower case `by`
2) `older than` might be better than `less than` since it's a datetime comparison.